Crashlytics/Crashlytics/Controllers/FIRCLSReportManager.m

```diff
@@ -207,6 +218,23 @@ - (instancetype)initWithManagerData:(FIRCLSManagerData *)managerData
   return [FBLPromise race:@[ collectionEnabled, _reportActionProvided ]];
 }
 
+/*
+ * This method returns a promise that is resolved once
+ * MetricKit diagnostic reports have been received by `metricKitManager`.
+ */
+- (FBLPromise *)waitForMetricKitData {
+  // If the platform is not iOS or the iOS version is less than 15, immediately resolve the promise
+  // since no MetricKit diagnostics will be available.
+#if CLS_METRICKIT_SUPPORTED
+  if (@available(iOS 15, *) && _settings.metricKitCollectionEnabled) {
```

Comment:
Same here

Reply:
Do we have any debug logs that print when MetricKit is just enabled? Might be good to put one here just so we know when customers write in

Reply:
Would it be better to put the log where we initially initialize the metrickit manager?

Reply:
Put it in the initial part (can move later if needed).